Automattic/newspack-plugin
# fix: correct text domains for popup wizard

### All Submissions:


### Changes proposed in this Pull Request:

This PR corrects the text-domain in the assets/wizards/popups directory.

See 1200550061930446-as-1205509524123559

### How to test the changes in this Pull Request:

1. Spot check the changes in the PR and confirm they're just changing the text domain on translateable strings to `newspack-plugin`.
2. Run `npm run build` on this branch, and spot-check the Campaign wizard screens.

### Other information:
